core/src/main/java/org/apache/calcite/sql2rel/StandardConvertletTable.java

```diff
@@ -260,11 +258,11 @@ public StandardConvertletTable(StandardConvertletTableConfig config) {
     registerOp(SqlStdOperatorTable.FLOOR, floorCeilConvertlet);
     registerOp(SqlStdOperatorTable.CEIL, floorCeilConvertlet);
 
-    registerOp(SqlStdOperatorTable.TIMESTAMP_ADD, new TimestampAddConvertlet());
-    if (config.shouldDecomposeTimestampdiff()) {
-      registerOp(SqlStdOperatorTable.TIMESTAMP_DIFF,
-          new TimestampDiffConvertlet());
-    }
+//    registerOp(SqlStdOperatorTable.TIMESTAMP_ADD, new TimestampAddConvertlet());
+//    if (config.shouldDecomposeTimestampdiff()) {
+//      registerOp(SqlStdOperatorTable.TIMESTAMP_DIFF,
+//          new TimestampDiffConvertlet());
+//    }
```

Comment:
Of note: We actually added the `config.shouldDecomposeTimestampdiff()`, this was our change. So that we don't have to comment out all the Calcite tests, my vote is to change this to: 

```suggestion
//    if (config.shouldDecomposeTimestampdiff()) {
//      registerOp(SqlStdOperatorTable.TIMESTAMP_DIFF,
//          new TimestampDiffConvertlet());
//    registerOp(SqlStdOperatorTable.TIMESTAMP_ADD, new TimestampAddConvertlet());
//    }
```

That way, we don't have to comment out all the Calcite tests/code.